Sources/Datadog/RUM/Instrumentation/Actions/UIKit/UIKitRUMUserActionsHandler.swift

```diff
@@ -37,6 +38,7 @@ internal class UIKitRUMUserActionsHandler: UIEventHandler {
 
     func publish(to subscriber: RUMCommandSubscriber) {
         self.subscriber = subscriber
+        rageActionManager.subscriber = subscriber
     }
```

Comment:
We can remove the `subscriber` and let the `RageTapManager` handle all commands.